walletlibrary/src/main/java/com/microsoft/walletlibrary/wrapper/OpenIdResolver.kt

```diff
@@ -36,6 +37,23 @@ object OpenIdResolver {
         }
     }
 
+    internal suspend fun validateRequest(requestContent: PresentationRequestContent): OpenIdRawRequest {
+        when (val presentationRequestResult =
+            VerifiableCredentialSdk.presentationService.validateRequest(requestContent)) {
+            is Result.Success -> {
```

Comment:
this method and lines 25 - 36ish are the same. It might be cleaner to have a private fun for this logic?